core/src/test/java/com/google/cloud/sql/core/CoreSocketFactoryTest.java

```diff
@@ -114,7 +114,7 @@ public class CoreSocketFactoryTest {
   private static final String SERVER_MESSAGE = "HELLO";
 
   private static final String PUBLIC_IP = "127.0.0.1";
-  private static final String PRIVATE_IP = "127.0.1.1";
+  private static final String PRIVATE_IP = "127.0.0.2";
```

Comment:
nit: can we add a comment that on macos, this command needs to be run to enable the tests to run?